### DISCUSSION TITLE
unit constraint setting

### DISCUSSION THREAD

**Discussion by RowanPeng** [spine-tools/SpineOpt.jl#559] [Ideas]
Hi,

I am trying to consturct a very basic load model to get familiar with SpineOpt. But here is one problem that I was stucked on. I was trying to enable the "windfarm" unit to have minimum capacity, if the generation is below the constraint, then will be shut down. Could you help me with this, please? I am confused by wich parameter_name to use for acheiving this. If there are any reference that will enable me to get to know more about all the parameters_name and some descriptions about thoes will be more helpful. 

Thank you.

Regards,

Peng

**Comment by mihlema** [comment]
Hi Peng,

thanks for opening this discussion. I am not sure if you have already found the documentation on SpineOpt:
![image](https://user-images.githubusercontent.com/40030652/223283975-5a4ac3ed-0fce-4fe2-8501-921ecbbcde1b.png)
On the SpineOpt github page, you can click on the ` docs`  icon, and you will be redirected to the documentation of SpineOpt.

That being said, the documentation for SpineOpt is not perfect, and possibly you still have questions. This [issue](https://github.com/spine-tools/SpineOpt.jl/issues/473) is the "landing page"  for new users, linking to our monthly calls, where everyone can get together and discuss issues, ideas etc. We also have gitter for messaging (see the small icon `chat` in the screenshot).

Not knowing your problem indetail, I think you probably want to use the parameter [minimum_operating_point](https://spine-tools.github.io/SpineOpt.jl/latest/concept_reference/Parameters/#minimum_operating_point) (defined in your case on probably a ` unit__to_node`  relationship), in combination with [online_variable_type](https://spine-tools.github.io/SpineOpt.jl/latest/concept_reference/Parameters/#online_variable_type) (to enforce on-off behavior of your unit).

Hopes this helps, feel free to join our meeting tomorrow!
